import-export-cli/utils/tokenManagement.go

```diff
@@ -313,7 +313,8 @@ func GetBase64EncodedCredentials(key, secret string) (encodedValue string) {
 func GetOAuthTokens(username, password, b64EncodedClientIDClientSecret, url string) (map[string]string, error) {
 	validityPeriod := DefaultTokenValidityPeriod
 	body := "grant_type=password&username=" + username + "&password=" + password + "&validity_period=" +
-		validityPeriod + "&scope=apim:api_view+apim:app_import_export+apim:app_owner_change+apim:subscribe+apim:api_publish"
+		validityPeriod + "&scope=apim:api_view+apim:app_import_export+apim:app_owner_change+apim:subscribe" +
```

Comment:
with password grant type this parameter will not be use i meant the validityPeriod.

Reply:
yes. removed the parameter with https://github.com/wso2/product-apim-tooling/pull/104/commits/2c76e291bbd3ed9e09ea496811caba97d9695b57